### DISCUSSION TITLE
Request to upload torchscript model

### DISCUSSION THREAD

**Discussion by csukuangfj** [speechbrain/speechbrain#2359] [Feature Request]
### 🚀The feature

https://huggingface.co/speechbrain/asr-transformer-transformerlm-librispeech/tree/main

provides only `model.state_dict()`.

Is it possible to also upload models after applying `torch.jit.script()` or `torch.jit.trace()`?

One of the advantages is that users don't need to install speechbrain to use the uploaded model.

### Solution outline

Upload models after applying `torch.jit.script()` or `torch.jit.trace()`.

### Additional context

We are going to use k2 for CTC decoding for any models trained with CTC loss.
See https://github.com/k2-fsa/k2/pull/1096

We want to load a model using `torch.jit.load()` from some framework without the need to install the framework where the model is trained.

**Comment by Adel-Moumen** [comment]
Hello guys @TParcollet @mravanelli, could you please have a look? thanks :-)
